utils/firestore.js

```diff
@@ -2185,10 +2202,15 @@ const queryKitsByReceivedDate = async (receivedDateTimestamp) => {
 
 const eligibleParticipantsForKitAssignment = async () => {
     try {
-        const { collectionDetails, baseline, bioKitMouthwash, kitStatus } = fieldMapping;
+        const { addressPrinted, collectionDetails, baseline, bioKitMouthwash, bloodOrUrineCollectedTimestamp, kitStatus } = fieldMapping;
+
+        const snapshot = await db.collection("participants")
+            .select(...participantHomeCollectionKitFields)
+            .where(`${collectionDetails}.${baseline}.${bioKitMouthwash}.${kitStatus}`, '==', addressPrinted)
+            .orderBy(`${collectionDetails}.${baseline}.${bloodOrUrineCollectedTimestamp}`)
+            .get();
 
-        const snapshot = await db.collection("participants").where(`${collectionDetails}.${baseline}.${bioKitMouthwash}.${kitStatus}`, '==', fieldMapping.addressPrinted).get();
-        if (snapshot.size !== 0) return snapshot.docs.map(doc => processParticipantData(doc.data(), false));
+        if (snapshot.size !== 0) return snapshot.docs.map(doc => processParticipantHomeMouthwashKitData(doc.data(), false));
         else return false;
```

Comment:
Should it return an empty array to make data type consistent?

Reply:
Yes, was trying to touch as little of his code as possible. Updated.

Reply:
Thanks. I usually do cleanups during implementation.